unleash-yggdrasil/src/strategy_upgrade.rs

```diff
@@ -158,11 +158,13 @@ fn upgrade_strategy(
 }
 
 fn upgrade_flexible_rollout_strategy(strategy: &Strategy) -> String {
-    let rollout = strategy.get_param("rollout");
+    let rollout = strategy
```

Comment:
Validates that our rollout parameters are actually a numeric type. This gets its own code because in flexible rollout we use a completely different parameter name